tapo/src/api/child_devices/ke100_handler.rs

```diff
@@ -0,0 +1,176 @@
+    ///
+    /// # Arguments
+    ///
+    /// * `min_temperature` - between 5 and 15
+    pub async fn set_min_temperature(&self, min_temperature: u8) -> Result<(), Error> {
+        let json = serde_json::to_value(TrvSetDeviceInfoParams::new().min_temp(min_temperature)?)?;
+        let request = TapoRequest::SetDeviceInfo(Box::new(TapoParams::new(json)));
+
+        self.hub_handler
+            .control_child::<serde_json::Value>(self.device_id.clone(), request)
+            .await?;
+
+        Ok(())
+    }
+
+    /// Sets the *minimal control temperature*.
+    ///
+    /// # Arguments
+    ///
+    /// * `min_control_temperature`
+    pub async fn set_min_control_temperature(&self, min_control_temperature: u8) -> Result<(), Error> {
+        let json = serde_json::to_value(TrvSetDeviceInfoParams::new().min_control_temp(min_control_temperature)?)?;
+        let request = TapoRequest::SetDeviceInfo(Box::new(TapoParams::new(json)));
+
+        self.hub_handler
+            .control_child::<serde_json::Value>(self.device_id.clone(), request)
+            .await?;
+
+        Ok(())
+    }
+
+    /// Sets the *maximum control temperature*.
+    ///
+    /// # Arguments
+    ///
+    /// * `max_control_temperature`
+    pub async fn set_max_control_temperature(&self, max_control_temperature: u8) -> Result<(), Error> {
+        let json = serde_json::to_value(TrvSetDeviceInfoParams::new().max_control_temp(max_control_temperature)?)?;
+        let request = TapoRequest::SetDeviceInfo(Box::new(TapoParams::new(json)));
+
+        self.hub_handler
+            .control_child::<serde_json::Value>(self.device_id.clone(), request)
+            .await?;
+
+        Ok(())
+    }
+
+    /// Sets frost protection on the device to *on* or *off*.
+    ///     
+    /// # Arguments
+    ///
+    /// * `frost_protection_on` - true/false
+    pub async fn set_frost_protection(&self, frost_protection_on: bool) -> Result<(), Error> {
+        let json = serde_json::to_value(TrvSetDeviceInfoParams::new().frost_protection_on(frost_protection_on)?)?;
+        let request = TapoRequest::SetDeviceInfo(Box::new(TapoParams::new(json)));
+    
+        self.hub_handler
+            .control_child::<serde_json::Value>(self.device_id.clone(), request)
+            .await?;
+
+        Ok(())
+    }
+
+    /// Sets child protection on the device to *on* or *off*.
+    ///     
+    /// # Arguments
+    ///
+    /// * `child_protection_on` - true/false
+    pub async fn set_child_protection(&self, child_protection_on: bool) -> Result<(), Error> {
+        let json = serde_json::to_value(TrvSetDeviceInfoParams::new().child_protection(child_protection_on)?)?;
+        let request = TapoRequest::SetDeviceInfo(Box::new(TapoParams::new(json)));
+    
+        self.hub_handler
+            .control_child::<serde_json::Value>(self.device_id.clone(), request)
+            .await?;
+
+        Ok(())
+    }
+
+    /// Sets the *temperature offset*.
```

Comment:
In case you have the device (and thus the radiator) close to a window, the temperature measured at the device itself may not reflect the actual room temperature. Thus you can set an offset to align with your room temperature.

Reply:
Makes sense. Thank you!